metafix/src/main/java/org/metafacture/metafix/FixMethod.java

```diff
@@ -106,6 +125,7 @@ public void apply(final Metafix metafix, final Record record, final List<String>
         }
     },
     array { // array-from-hash
+
```

Comment:
There are empty lines added across several files. Was this intentional?

Reply:
Uh, some more I overlooked :/  Fixed.